docs/transactions.asciidoc

```diff
@@ -1,48 +1,40 @@
 [[transactions]]
 == Transactions
 
-A transaction represents one event, captured by an Elastic APM agent within one service. 
-A transaction can for example be a single HTTP request or an asynchrounous background job within one service.
+A transaction describes an event captured by an Elastic APM agent instrumeting a service.
+A transaction can for example be a HTTP request or an asynchrounous background job.
 
-The transaction shows the duration of the event, 
-a unique id, the type and an automatically retrieved name,
-as well as an indication whether or not the transaction was handled successfully. 
-
-
-If a transaction was sampled, 
-the <<transaction-spans,spans>> of the transaction were captured and are available as seperate documents.
-There is a configuration option on how many spans should be captured per transaction.
-
-A transaction can also contain information regarding `marks`. 
-Marks capture the timing in milliseconds of a significant event during the lifetime of a transaction, 
-set by the user or the agent
+It contains the timestamp and duration of the event,
+a unique id, type, name,
+a result (e.g. response code), some contextual data, and other relevant information depending on the agent.
+For instance, the JavaScript RUM captures transaction marks,
+which are points in time relative to the start of the transaction with some label.
 
 [[transactions-context]]
 include::./context.asciidoc[]
 
+Agents decide whether to sample transactions or not, and provide settings to control sampling behaviour.
+If sampled,
+the <<transaction-spans,spans>> of a transaction are sent and stored seperate documents.
+
+Transactions are stored in <<transaction-indices, transaction indices>>.
+
 [[transaction-spans]]
 [float]
 === Spans
-Several spans can be related to a transaction. 
-The relationship is indicated by a `transaction.id`.
-
-A span contains information about a specific code path, 
-executed as part of a transaction. 
-Elastic APM agents automatically instrument a variety of libraries, 
-but also support custom instrumentation for code paths.
-Every code path that is captured by an agent creates a span.
+Spans can have 0, 1 or many spans. Spans have a `transaction.id` attribute that refer to their transaction.
```

Comment:
minor: Just an FYI that use the oxford comma at Elastic: So this would be "0, 1, or many spans"